### ISSUE TITLE
Critical Error in 1.0.1

### ISSUE THREAD

**Issue by lane-c-wagner** [koca/vue-prism-editor#83]
Take a look at https://classroom.qvault.io/playground/go

Try to write code longer than ~26 lines. You will see that the cursor is no longer able to move downwards. Not sure what is causing this but I will likely have to rollback until a solution is pushed out

The problem is also present in your example: https://codesandbox.io/s/61yrlnlnmn
However, there it starts around line ~16

It's looking like it's dependent on the height of the <textarea>. It isn't resizing dynamically or something...

**Comment by lane-c-wagner** [comment]
Should be an easy fix. I was able to patch it with the following CSS:

```css
.prism-editor-wrapper {
  height: auto;
}
```

**Comment by koca** [comment]
Oh! thanks for the report @lane-c-wagner. Let me check.

**Comment by lane-c-wagner** [comment]
I patched qvault, so you will need to verify on your demo. Sorry above t that haha

**Comment by koca** [comment]
Made `width` and `height` auto and didn't fix 🤔 here is a repro:
https://codesandbox.io/s/vue-prism-editor-bug-83-xdpt1?file=/src/App.vue
I added the css of the editor so you can play with it.

**Comment by lane-c-wagner** [comment]
Adding `!important` to those additions you made fixed it. I don't think that the `!important` is the right choice though, probably just need to also change your duplicate rules:

```
.prism-editor-wrapper .prism-editor__textarea {
  position: absolute;
  top: 0;
  left: 0;
  height: 100%;
  width: 100%;
  resize: none;
  color: inherit;
  overflow: hidden;
  -moz-osx-font-smoothing: grayscale;
  -webkit-font-smoothing: antialiased;
  -webkit-text-fill-color: transparent;
}
```

or

```
.prism-editor-wrapper .prism-editor__container {
  position: relative;
  text-align: left;
  box-sizing: border-box;
  padding: 0;
  overflow: hidden;
  width: 100%;
  height: 100%;
}
```

**Comment by koca** [comment]
I think its a issue with `display: flex` on `.prism-editor-wrapper`

can you revert your changes and try with `max-width:none` & `min-width:none` like this: 
```css
.prism-editor-wrapper {
  max-width: none;
  max-height: none;
```

**Comment by koca** [comment]
Removing this:
https://github.com/koca/vue-prism-editor/blob/2f161603b5785da546ff85a521dfad3bf4503db8/packages/vue-prism-editor/src/styles.css#L27

fixes all cases i think. Would you mind trying and updating the PR?

**Comment by lane-c-wagner** [comment]
@koca done, and thanks

**Comment by koca** [comment]
released `v1.2.0` 🚀
